src/bin/agent/start.ts

```diff
@@ -15,6 +15,7 @@ const start = binUtils.createCommand('start', {
   passwordFile: true,
 });
 start.option('-b, --background', 'Starts the agent as a background process');
+start.option('-t --test', 'starts and stops the agent as a quick sanity test.');
```

Comment:
Should replace this with a background and then in the gitlab YAML directly do a `pk agent stop`. You may need to "sleep" for 1 second.

Reply:
I think it's better to have the test option in agent start but I can remove the flag and have the option hidden as an ENV variable. the test option would also be a very useful debug command down the line.

starting the agent in the background and stopping it complicates the test script a fair bit and I think would be unreliable.

Reply:
Ok, a sleep might be flaky.

Let's think about the agent testing holistically later. For now a simple sanity check is fine. So running the help page is good enough.

But adding a `--test` flag is not a good idea. It doesn't fit in the rest of the API.

Reply:
I can change it so we can only set it with an env to make it semi-hidden.

Reply:
It's just running the help page now.